docs/build/tooling/hardhat/hardhat-zksync-solc.md

```diff
@@ -109,6 +109,34 @@ For Solidity versions older than 0.8, only this compilation mode is available an
 
 :::
 
+### zkSync Era Solidity compiler
+
+Due to the identified limitations in the [Solidity compiler](https://docs.zksync.io/zk-stack/components/compiler/toolchain/solidity.html#limitations), our team has developed a new [zkSync Era compiler](https://github.com/matter-labs/era-solidity) known as EraVM, which effectively addresses and resolves these constraints.
```

Comment:
@hedgar2017 Is `https://github.com/matter-labs/era-solidity` the correct GitHub repo to reference here?

Reply:
or should it be `https://github.com/matter-labs/era-compiler-solidity`?

Reply:
@MexicanAce it's the 1st one, but I would rewrite this paragraph from scratch:

Due to [the identified limitations](https://docs.zksync.io/zk-stack/components/compiler/toolchain/solidity.html#limitations) of the [upstream Solidity compiler](https://github.com/ethereum/solidity), our team has developed [a new edition](https://github.com/matter-labs/era-solidity) of the compiler, which effectively addresses and resolves these constraints.

Reply:
Thanks @hedgar2017 for the suggestion. In the most recent commit, I have changed the paragraph accordingly with your recommendation.